app/models/device.rb

```diff
@@ -1,8 +1,8 @@
 class Device < ApplicationRecord
   belongs_to :user
+  validates :user_id, presence: true
```

Comment:
should we undo the diff here? move this back down to keep all the validations together?

Reply:
Yes, missed this